src/lib/message/L1ToL2MessageCreator.ts

```diff
@@ -164,7 +164,7 @@ export class L1ToL2MessageCreator {
         )
 
     const tx = await this.l1Signer.sendTransaction({
-      ...createRequest,
+      ...createRequest.txRequest,
```

Comment:
I added the property to the spread object to be consistent with how the rest of methods make these calls (they all add the property to the spread object, instead of destructuring them)

Reply:
i mean on line 157 it can be 
`const { txRequest } = isL1ToL2TransactionRequest(params) ...`

Reply:
Sorry, what I mean is that other methods like `deposit` and `withdraw` of both `ethBridger` and `assetBridger` have a similar behaviour (create the tx request and then sendTransaction), and they don't destructure the object but just add `txRequest` at the end. That's the convention I tried to follow.

Reply:
sounds good, thanks for the clarification!